terra/compute/docker.py

```diff
@@ -162,11 +165,4 @@ class Service(ContainerService):
   Base docker service class
   '''
 
-  def __init__(self):
-    super().__init__()
-
-    # default docker-compose file (if file exists)
-    compose_file = os.path.join(self.env['TERRA_APP_DIR'],
-                                'docker-compose.yml')
-    if os.path.isfile(compose_file):
-      self.compose_files = [compose_file]
+  compose_files = []
```

Comment:
The defaut compose_file is made use of in several terra_dsm apps - why does this feature need to be removed?

Reply:
Darn. Ok, I'll put it back then